FirebaseAuth/Sources/Swift/Utilities/AuthRecaptchaVerifier.swift

```diff
@@ -0,0 +1,194 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if os(iOS)
+
+  import Foundation
+  import RecaptchaInterop
+
+  @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
```

Comment:
Note, this file is conditionally compiled for iOS but the types have availability attributes for other platforms.

Reply:
Understood. I could be convinced otherwise, but I've been using a consistent `@available` for async/await checking.

Reply:
I don't feel strongly either way. I wouldn't be opposed to keeping the same consistent `@available` for copy/paste convenience, if that's easier.